### PR TITLE
Add required

### PR DESCRIPTION
**Pull request by garcha**
- [x] Updated README
- [x] Changed branch to develop

### CODE REVIEW

File: src/GooglePlacesAutocomplete/index.js [hunk 1]
```diff
@@ -353,6 +356,7 @@ GooglePlacesAutocomplete.defaultProps = {
     container: {},
     suggestion: {},
   },
+  required: false
```
Comment (Tintef) [inline src/GooglePlacesAutocomplete/index.js]: Missing trailing comma

File: src/GooglePlacesAutocomplete/index.js [hunk 2]
```diff
@@ -331,6 +333,7 @@ GooglePlacesAutocomplete.propTypes = {
   renderSuggestions: PropTypes.func,
   suggestionsClassNames: suggestionClassNamesType,
   suggestionsStyles: suggestionStylesType,
+  required: PropTypes.bool
```
Comment (Tintef) [inline src/GooglePlacesAutocomplete/index.js]: Missing trailing comma

**Comment by garcha** [comment]
added the commas

**Comment by garcha** [comment]
thanks for merging this in!

**Pull request merged**